bench/worker/worker_bench.rb

```diff
@@ -55,6 +56,15 @@ def dup_resque_redis
   end
 
   def run
+    n_reports = [*@nreports.strip.split(',')].map(&:to_i)
+
+    # Warming up...
+    run_benchark(10000, print_report: false)
+
+    n_reports.each { run_benchark(_1) }
```

Comment:
this is fine, just a nitpick, perhaps output has multiple times `    "      user     system      total        real\n"` (this is `CAPTION`) instead of the original table. But it is a minor thing so I don't care too much. Just saying.

Reply:
Yes, it prints a different table for each batch. Is that what you don't like?

I can refactor it to show only one table with all results.

Reply:
It makes sense to be a single table but as I said, not super important.

Reply:
Done in https://github.com/3scale/apisonator/pull/403/commits/2a7018873a3d54cde6565194bf38ad2b982f827e

Now it looks like this:

```
$ NUM_REPORTS=1000,1000,1000 bundle exec rake bench[worker/worker_bench.rb]
Running benchmark /home/jlledom/redhat/apisonator/bench/worker/worker_bench.rb
generating 10000 transactions..
Queue has 12000 jobs..
Shutting down worker..
generating 1000 transactions..
Queue has 1200 jobs..
Shutting down worker..
generating 1000 transactions..
Queue has 1200 jobs..
Shutting down worker..
generating 1000 transactions..
Queue has 1200 jobs..
Shutting down worker..
======================================================================
      user     system      total        real
  0.508081   0.067297   0.575378 (  1.776290) 48580KB (1000 reports)
  0.444679   0.070898   0.515577 (  1.707208) 48836KB (1000 reports)
  0.463991   0.059394   0.523385 (  1.651240) 48836KB (1000 reports)
======================================================================
Benchmarks finished
```